scripts/client_test/cli_test.py

```diff
@@ -273,16 +273,14 @@ def test_all(self) -> None:
             self.build_runtime(rt["workdir"])
 
         for name, expl in EXAMPLES.items():
-            p = subprocess.Popen(
+            logger.info(f"preparing data for {expl}")
+            rc = subprocess.call(
                 ["make", "CN=1", "prepare-data"],
-                stdout=subprocess.PIPE,
-                stderr=subprocess.STDOUT,
                 cwd=expl["workdir"],
             )
```

Comment:
https://docs.python.org/3/library/subprocess.html#subprocess.Popen.wait
> Note This will deadlock when using stdout=PIPE or stderr=PIPE and the child process generates enough output to a pipe such that it blocks waiting for the OS pipe buffer to accept more data. Use [Popen.communicate()](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate) when using pipes to avoid that.